esp/esp/program/modules/apps.py

```diff
@@ -2,3 +2,9 @@
 
 class ModulesConfig(InstallConfig):
     name = 'esp.program.modules'
+
+    def ready(self):
+        super(ModulesConfig, self).ready()
+        # TODO(benkraft): add a thing to InstallConfig that imports a signals
```

Comment:
Not all of our apps use InstallConfig, that one is specifically for apps that have an install(). If we wanted to refactor to have all our apps use some subclass of AppConfig that runs install() if it exists and imports .signals if it exists, that's a thing we could do.